### ISSUE TITLE
Speed Controller not responding on videos?

### ISSUE THREAD

**Issue by Supperz1** [igrigorik/videospeed#395]
I've been using Video Speed for ages without a problem. I just got a new laptop, and after logging in to all my Chrome accounts, the Video Speed Controller does not respond on any video it would otherwise (ie Youtube, Facebook videos, etc.). The bar itself is still visible and all the buttons are clickable, but don't do anything. Any possible solution? I downloaded Flash player, if it's relevant.

**Comment by bethrobson** [comment]
Yep I'm seeing this too - stopped working in Chrome. Still working in Firefox.

**Comment by yambu** [comment]
same for me
